src/sentry/api/endpoints/organization_events.py

```diff
@@ -72,6 +72,10 @@ def get(self, request: Request, organization) -> Response:
 
         sentry_sdk.set_tag("performance.metrics_enhanced", metrics_enhanced)
         allow_metric_aggregates = request.GET.get("preventMetricAggregates") != "1"
+
+        query_modified_by_user = request.GET.get("user_modified")
+        if query_modified_by_user is not None:
```

Comment:
lets validate the value too, so we don't get garbage in the tag eg.
```suggestion
        if query_modified_by_user is in ["True", "False"]:
```